angular/build/config/vendor.yml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
plugins.yml (the gitignored file in the same folder) is auto generated
based on where you do

```
plugin.use_component
# or
plugin.use_asset
```

in any plugin files

On Wed, Feb 17, 2016 at 4:57 PM, James Kiesel <james.kiesel@gmail.com>
wrote:

> Vendor is hand written; has to be, because we have no way of automatically
> knowing what files from within the node_modules to include
>